todo_app/e2e_test.py

```diff
@@ -6,55 +6,54 @@
 from flask import Flask
 import os
 from todo_app.app import create_app
-from dotenv import load_dotenv,find_dotenv
+from dotenv import load_dotenv, find_dotenv
 import time
 
 
-
-
-
 def create_trello_board():
-    
+
     key = os.environ['KEY']
     token = os.environ['TOKEN']
-    board_response = requests.post(f'https://api.trello.com/1/boards/?name=test&key={key}&token={token}')
+    board_response = requests.post(
+        f'https://api.trello.com/1/boards/?name=test&key={key}&token={token}')
     board_id = board_response.json()["id"]
-    response =  requests.get(f'https://api.trello.com/1/boards/{board_id}/lists?key={key}&token={token}')
-    
-	
+    response = requests.get(
+        f'https://api.trello.com/1/boards/{board_id}/lists?key={key}&token={token}')
+
     lists = response.json()
-    
+
     for list in lists:
-        
+
         if list['name'] == "TODO":
             os.environ['TOID'] = list['id']
         elif list['name'] == "DOING":
             os.environ['DOINGID'] = list['id']
         elif list['name'] == "DONE":
             os.environ['DONE'] = list['id']
     return board_id
-    #return application
+    # return application
+
 
 def delete_trello_board(board_id):
-     key = os.environ['KEY']
-     token = os.environ['TOKEN']
-     board_delete = requests.delete(f'https://api.trello.com/1/boards/{board_id}?key={key}&token={token}')
-   
-    
+    key = os.environ['KEY']
+    token = os.environ['TOKEN']
+    board_delete = requests.delete(
+        f'https://api.trello.com/1/boards/{board_id}?key={key}&token={token}')
+
+
 @pytest.fixture(scope='module')
 def app_with_temp_board():
-# Create the new board & update the board id environment variable
-    
-   
-# construct the new application
+    # Create the new board & update the board id environment variable
+
+    # construct the new application
     file_path = find_dotenv('.env')
     load_dotenv(file_path, override=True)
     board_id = create_trello_board()
```

Comment:
You don't need to create and delete Trello boards any more. Instead use a different MongoDB database or collection for the test and then drop it afterwards.